tests/test_dog_arguments.py

```diff
@@ -54,63 +59,65 @@ def test_as_root(call_centos7, capfd):
 
 
 def test_pull_as_echo_argument(call_centos7, capstrip):
-    '''--pull should only be interpreted as an dog argument if it comes before the first argument'''
+    # --pull should only be interpreted as a dog argument if it comes before the first
+    # argument
     call_centos7('echo', '-n', '--pull')
     captured = capstrip.get()
     assert captured == ('--pull', '')
 
 
 def test_version(call_dog, capfd):
-    '''--version should just return the current dog version'''
+    # --version should just return the current dog version
     call_dog('--version')
     captured = capfd.readouterr()
     assert re.match('dog version [0-9]+', captured.out)
 
 
 def test_verbose(call_centos7, capfd):
-    '''--verbose should report the actual setup'''
+    # --verbose should report the actual setup
     call_centos7('--verbose', 'id')
     captured = capfd.readouterr()
     assert 'Dog Config' in captured.out
     assert "'verbose': True" in captured.out
 
 
 def test_stdin_testing_works(call_shell, capstrip):
-    '''Just verifying that my stdin testing works before testing it with dog.'''
+    # Just verifying that my stdin testing works before testing it with dog.
```

Comment:
Is this a new way of writing docstrings? - if that is the case then i think we should use the same syntax all over the place.